src/components/app/TabBar.js

```diff
@@ -51,6 +52,21 @@ class TabBar extends PureComponent {
   }
 }
 
+type Props = {
+  className?: string,
+  tabs: Tab[],
+  selectedTabName: string,
+  tabOrder: number[],
+  onSelectTab: string => mixed,
+  onChangeTabOrder: (number[]) => any,
+  +_mouseDownListener?: () => mixed,
```

Comment:
`_mouseDownListener` didn't belong in `Props` at all. More nonsense from before I figured out the bound member function stuff (another thing I'd be curious to have explained).